backend/app/Services/WorkOrder/WorkOrderService.php

```
@@ -48,10 +46,260 @@ public function createWorkOrder(array $data): WorkOrder
+     * Explicit ids win (deduplicated, order preserved); otherwise fall back to
+     * the single highest-version active template for the product type.
+     *
+     * @param  array<int, int|string>  $templateIds
+     * @return EloquentCollection<int, ProcessTemplate>
+     */
+    protected function resolveBomTemplates(?int $productTypeId, array $templateIds): EloquentCollection
+    {
+        $ids = $this->normalizeIds($templateIds);
+
+        if (! empty($ids)) {
+            $found = ProcessTemplate::whereIn('id', $ids)->get()->keyBy('id');
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\ProcessTemplate::whereIn().


(staticMethod.notFound)